pkg/controllers/apps/aggregatestatus/aggregatestatus.go

```diff
@@ -292,6 +292,7 @@ func (c *Controller) updateSubscriptionStatus(sub *appsapi.Subscription, status
 	// Or create a copy manually for better performance
 
 	klog.V(5).Infof("try to update Subscription %q status", sub.Name)
+	sub = sub.DeepCopy()
```

Comment:
Here `sub` is a pointer. Better use another object instead of `sub`.

Reply:
ok, I choose another object and commit